database_schemas/cassandra/benchflow.cql

```diff
@@ -75,33 +74,35 @@ CREATE TABLE network_interface_data (
   network_tx_bytes bigint,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), container_id, host_id, network_interface_name, network_interface_data_id)
+  PRIMARY KEY ((experiment_id, trial_id), container_id, host_id, network_interface_name)
 );
 
 CREATE TABLE process (
-  process_instance_id uuid,
+  process_instance_id text,
   source_process_instance_id text,
   process_definition_id text,
   start_time timestamp,
   duration bigint,
   end_time timestamp,
+  to_ignore boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), process_instance_id)
+  PRIMARY KEY ((experiment_id, trial_id), process_instance_id)
 );
 
 CREATE TABLE construct (
-  construct_instance_id uuid,
+  construct_id text,
   source_construct_instance_id text,
   construct_type text,
   construct_name text,
   start_time timestamp,
   duration bigint,
   end_time timestamp,
   process_instance_id uuid,
+  to_ignore boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), construct_instance_id, process_instance_id)
+  PRIMARY KEY ((experiment_id, trial_id), process_instance_id, source_construct_instance_id, construct_id)
```

Comment:
Why do you also have `source_construct_instance_id` here and not `source_process_instance_id` in `process` table?